assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,26 @@
+// ----- Flex-box mixin (work in progress)
+@mixin grid($flex, $direction:column) {
+  flex-direction: $direction;
+  display:flex;
+  
+  & > li {      
+	  flex:$flex;
+    list-style:none;
+    margin:0 1em 0 0;
+    padding: 1em;
+  }
+  
+  @media all and (min-width: 480px) {
+  & {flex-direction: row;}  
+  }
+}
+
```

Comment:
Recognize the mixin as a WIP, but it also feels like a premature abstraction. For example, we cannot always assume that there will be an `li` in the structure. 

First get it working, then see if we can abstract it out into a reusable tool. I'm super guilty of abstracting early on. Been learning the value of iterating quickly and abstracting later.